The python_cmd variable below gets inserted into Python wrapper scripts
installed by this package. For text files on Windows, Conda's prefix detection
looks for a "mixed" form path ("C:/conda/conda-bld/...."), so we need to
express python_cmd in this way in order for the prefix substitution to work in
these files. Note that in *binary* files Conda expects the Windows-native form
("C:\conda\conda-bld\...").

diff --git a/tools/meson.build b/tools/meson.build
index 1552dc6..0bc33ba 100644
--- a/tools/meson.build
+++ b/tools/meson.build
@@ -10,7 +10,7 @@ if get_option('doctool')
 endif
 
 if cc.get_id() == 'msvc'
-  python_cmd = '/usr/bin/env ' + python.get_variable('prefix') + '\\python.exe'
+  python_cmd = '/usr/bin/env ' + '/'.join(python.get_variable('prefix').split('\\')) + '/python.exe'
 else
   python_cmd = '/usr/bin/env python@0@'.format(python.language_version().split('.')[0])
 endif
